src/Features/Core/Portable/Completion/CompletionServiceWithProviders.cs

```diff
@@ -199,28 +200,28 @@ internal protected CompletionProvider GetProvider(CompletionItem item)
             var triggeredProviders = ImmutableArray<CompletionProvider>.Empty;
             switch (trigger.Kind)
             {
-                case CompletionTriggerKind.Insertion:
-                case CompletionTriggerKind.Deletion:
-                    if (this.ShouldTriggerCompletion(text, caretPosition, trigger, roles, options))
+            case CompletionTriggerKind.Insertion:
```

Comment:
Sure, but that's not the style elsewhere in Roslyn, and not what dotnet/codeformatter enforces.